src/components/AddBook.js

```diff
@@ -36,7 +36,7 @@ const AddBook = () => {
       <div className="add-container">
         <h2>ADD NEW BOOK</h2>
         <form>
-          <input type="text" placeholder="Book Title" onChange={handleTile} />
+          <input type="text" placeholder="Book Title" onChange={handleTitle} />
           <input type="text" placeholder="Book Author" onChange={handleAuthor} />
           <input type="text" placeholder="Category" onChange={handleCategory} />
           <button type="button" onClick={postBook}>Add Book</button>
```

Comment:
- [ **OPTIONAL** ] **RECOMMENDED:** Right now you can add and remove a book, but when it is added it is without **category**(the category property is empty), make sure you fix that issue. Here is the error showed:

| Error | Book object|
|--------- | -----------|
| ![Screenshot from 2022-07-30 20-33-03](https://user-images.githubusercontent.com/83042742/181936947-a839f492-dd42-43c2-88e4-e38183fa907f.png) | ![Screenshot from 2022-07-30 20-35-07](https://user-images.githubusercontent.com/83042742/181936995-b1612ec8-8f3c-441b-bef2-fd79b3c45405.png) |

Reply:
Okay thank you, I will fix the errors.